app/controllers/watson_controller.rb

```diff
@@ -0,0 +1,71 @@
+class WatsonController < ApplicationController
+    protect_from_forgery with: :null_session, if: Proc.new { |c| c.request.format == 'application/json' }
+  
+    def make_post_req  
+        puts "testing in process"
+        require 'net/http'
+        require 'uri'
+    
+        begin
+        
+            
+            url = URI("https://gateway.watsonplatform.net/natural-language-understanding/api/v1/analyze?version=2018-03-16")
+            
+            http = Net::HTTP.new(url.host, url.port)
+            http.use_ssl = true
+            http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+            
+            req = Net::HTTP::Put.new(url)
```

Comment:
Is it a Put or Post request?

Reply:
I have no idea. I tried a lot of codes (you can see it in the same file at the bottom another test) Maybe someone could help me on Monday.